src/js/actions/libraries.js

```diff
@@ -85,7 +90,7 @@ define(function (require, exports) {
                 return Promise.fromNode(function (cb) {
                     representation.updateContentFromPath(path, false, cb);
                 });
-            }).finally(function () {
+            }).then(function () {
```

Comment:
I guess this was intentional? Is the operation somehow ended if there is a failure?